sproutsocial/tailor
# [WEB-174] Allow for sharing of utility class in the select-multi component

## Details of Change

* Hacks the JS to allow for using the window object of the utilities and JS classes so we don't have to load on each component


## Relevant Links

- [Jira Card](https://sprout.atlassian.net/browse/WEB-174)
- [Test Site](https://justin-test.stagely.sproutsocial.com/?tailor=1)


### Related PRs

- https://github.com/sproutsocial/web-insights-application/pull/1249


## Steps to Test or Reproduce

1. Go to https://justin-test.stagely.sproutsocial.com/?tailor=1 and see that the utilities are being shared from the `window.utilities` object
